gravitee-apim-console-webui/src/management/api/design/design/management-api-design.component.ts

```diff
@@ -0,0 +1,319 @@
+          type: 'boolean',
+        },
+        headers: {
+          type: 'array',
+          title: 'Request Headers',
+          items: {
+            type: 'object',
+            title: 'Header',
+            properties: {
+              name: {
+                title: 'Name',
+                type: 'string',
+              },
+              value: {
+                title: 'Value',
+                type: 'string',
+              },
+            },
+          },
+        },
+        body: {
+          title: 'Request body',
+          type: 'string',
+          'x-schema-form': {
+            type: 'codemirror',
+            codemirrorOptions: {
+              lineWrapping: true,
+              lineNumbers: true,
+              allowDropFileTypes: true,
+              autoCloseTags: true,
+            },
+          },
+        },
+        specification: {
+          title: 'Transformation (JOLT Specification)',
+          type: 'string',
+          'x-schema-form': {
+            type: 'codemirror',
+            codemirrorOptions: {
+              lineWrapping: true,
+              lineNumbers: true,
+              allowDropFileTypes: true,
+              autoCloseTags: true,
+              mode: 'javascript',
+            },
+          },
+        },
+      },
+      required: ['url', 'specification'],
+    },
+    documentation:
+      '= Custom (HTTP)\n\n=== How to ?\n\n 1. Set `Polling frequency interval` and `Time unit`\n2. Set the `HTTP service URL`\n 3. If the HTTP service doesn\'t return the expected output, add a JOLT `transformation` \n\n[source, json]\n----\n[\n  {\n    "key": 1,\n    "value": "https://north-europe.company.com/"\n  },\n  {\n    "key": 2,\n    "value": "https://north-europe.company.com/"\n  },\n  {\n    "key": 3,\n    "value": "https://south-asia.company.com/"\n  }\n]\n----\n',
+  },
+];
+const PROPERTY_PROVIDER_TITLES = PROPERTY_PROVIDERS.reduce((map, provider) => {
+  map[provider.id] = provider.name;
+  return map;
+}, {} as Record<string, string>);
+const PROPERTY_PROVIDER_IDS = Object.keys(PROPERTY_PROVIDER_TITLES);
+const DYNAMIC_PROPERTY_SCHEMA = {
+  properties: {
+    enabled: {
+      type: 'boolean',
+      title: 'Enabled',
+      description: ' This service is requiring an API deployment. Do not forget to deploy API to start dynamic-properties service.',
+    },
+    trigger: {
+      type: 'object',
+      properties: {
+        rate: {
+          type: 'integer',
+          title: 'Polling frequency interval',
+        },
+        unit: {
+          type: 'string',
+          title: 'Time unit',
+          enum: ['SECONDS', 'MINUTES', 'HOURS'],
+        },
+      },
+      required: ['rate', 'unit'],
+    },
+    provider: {
+      type: 'string',
+      title: 'Provider type',
+      enum: PROPERTY_PROVIDER_IDS,
+      default: PROPERTY_PROVIDER_IDS[0],
+      'x-schema-form': {
+        titleMap: PROPERTY_PROVIDER_TITLES,
+      },
+    },
+  },
+  required: ['trigger', 'provider'],
+};
```

Comment:
Maybe we can move this into another file to avoid flooding the one of the component